### PR TITLE
Feature: Add rich message types (image and file)

### LINKED ISSUE #121: Feature: Support rich messages types in app

### PR DESCRIPTION
**Pull request by muhzi4u**
Fixes #121 

**Preview**

Screenshot 2020-04-08 at 1 56 58 PM

photo6084490515828550419

### CODE REVIEW

File: src/components/ChatMessage.js [hunk 1]
```diff
@@ -131,9 +213,72 @@ const MessageContentComponent = ({ themedStyle, message, type, showImage }) => {
     }
   };
 
+  const renderAttachment = () => {
```
Comment (pranavrajs) [inline src/components/ChatMessage.js:216]: Can you make it a separate component?

File: src/components/ChatMessage.js [hunk 2]
```diff
@@ -131,9 +213,72 @@ const MessageContentComponent = ({ themedStyle, message, type, showImage }) => {
...
+                  <CustomText style={themedStyle.filenameText}>
+                    {fileName.length < 30
+                      ? `${fileName}`
+                      : `...${fileName.substr(fileName.length - 15)}`}
+                  </CustomText>
+
+                  <TouchableOpacity
+                    onPress={() => showAttachment({ type: 'file', dataUrl })}>
+                    <CustomText style={themedStyle.downloadText}>
+                      Download
```
Comment (pranavrajs) [inline src/components/ChatMessage.js:267]: i18n

File: src/components/ConversationItem.js [hunk 3]
```diff
@@ -9,10 +10,56 @@ import { dynamicTime } from '../helpers/TimeHelper';
...
 
+const FilePreviewIcon = (style) => {
+  return <Icon {...style} name="file-text-outline" />;
+};
+
+const AttachmentItem = ({ themedStyle, unread_count, theme, attachment }) => {
+  const { file_type: fileType } = attachment;
+  return (
+    <Fragment>
+      {fileType === 'image' ? (
```
Comment (pranavrajs) [inline src/components/ConversationItem.js:25]: This if conditions can be separate components

**Comment by muhzi4u** [comment]
@pranavrajs Please check now.

**Pull request merged**